build.gradle

```diff
@@ -588,6 +588,7 @@ subprojects {
       from "$rootDir/LICENSE"
       from "$rootDir/NOTICE"
       from sourceSets.test.output
+      exclude 'junit-platform.properties'
```

Comment:
Could you please add comments for it?

Reply:
Thank you for the reminder, I added the corresponding comment, please review again

Reply:
We don't need to add jira, and the comments should explain "why" rather that "what it does"